lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -146,17 +148,23 @@ export class LhcPeriodsOverviewModel extends Observable {
     }
 
     /**
-     * Get filter related query params as single string
-     * @return {string} filter search query phrase
+     * Getter to lhc periods beam type filter model
+     * @return {TextTokensFilterModel} lhc periods beam type filter model
      */
-    getFilteringQueryParametersExpression() {
-        const namesFilters = this._namesFilterModel.normalized
-            .map((lhcPeriodName) => `filter[names][]=${lhcPeriodName}`);
-
-        const yearsFilters = this._yearsFilterModel.normalized
-            .map((year) => `filter[years][]=${year}`);
+    get beamTypesFilterModel() {
+        return this._beamTypesFilterModel;
+    }
 
-        return [namesFilters, yearsFilters].flat().join('&');
+    /**
+     * Get filter related query params
+     * @return {object} filter search query parameters
+     */
+    getFilteringQueryParameters() {
+        return {
+            'filter[names]': this._namesFilterModel.normalized,
+            'filter[years]': this._yearsFilterModel.normalized,
+            'filter[beamTypes]': this._beamTypesFilterModel.normalized,
+        };
```

Comment:
To be confirmed, but I think here you can directly return an object 
```js
{
    filter: {
        name: ...
        years: ...
        beamTypes: ...
    }
}
```

This is the whole idea of having a `FilteringModel` (coming soon) that will have a map of keys leading to filter model, then this will be automatized